docs/tokenomics/supply.md

```diff
@@ -15,39 +15,37 @@ title: 🏭 Supply
 
-:::tip
+## LAVA prioritizes revenue, not inflation
 
-Providers and validators can be separate entities.
+Most other blockchains are hyper-inflationary. They reward Validators and LAVA stakers by inflating token supply. 
 
-:::
+LAVA has no supply inflation. Instead of minting new LAVA to reward network contributors, contributors share the network revenue. Contributors include: Validators, LAVA stakers, Data Providers and LAVA restakers. 
 
-## 💵 LAVA token has no inflation — Rewards are sourced directly from network revenue
+On Lava: 
 
-Lava has a **fixed supply of 1,000,000,000**. Furthermore, Lava has developed a novel mechanism for bootstrapping demand for the network’s service i.e. RPC and APIs.
+- Validators and LAVA stakers do NOT earn rewards by minting new tokens. 
+- Instead, blockchains and apps pay the network to provide services such as RPC. 
+- Blockchains often pay in that chain’s native token.
+- Majority of these rewards go to Providers and LAVA restakers.
 
-### 🔄 **Provider Drop Mechanism**
+This is why staking rewards are “relatively” low on Lava. Majority of rewards go to the Providers and LAVA restakers who delegate to Providers.
 
-- **6.6% of the total supply** is distributed on a monthly basis over 4 years to Providers as a reward boost called a Provider Drop.
-- This reserve is a **fixed allocation of rewards** used to incentivize Providers to join in the early stages of Lava.
+## Provider Drops & LAVA burn 
 
-- Depending on paid demand for Lava services, **monthly distribution is variable**; more paid demand generally results in more reward boosts being distributed.
+Many marketplaces, such as Lava, face an initial problem: demand does not come without high supply, but supply does not come without high demand. This is the “chicken and egg” or “cold start” problem. Lava has a novel solution for this. 
 
-  - Any unused allocation from the previous month is burned.
+44 million LAVA (4.4% of total supply) is distributed to attract Providers when there are fewer Data Consumers in the early stages of the protocol. This allocation is called Provider Drops. 
 
-As the network draws more and more consumers, the need for the Drops will decrease, as Providers will make the difference from subscription payments.
+Provider Drops are distributed across 4 years and vary on a monthly basis depending on how much paid demand there is for RPC via Lava. The higher the demand, the more Providers earn from the Provider Drop allocation. If there is less demand, the Provider Drop allocation is not as needed and therefore burned. 
 
-| Parameter                                        | % of total `LAVA` supply |
-| ------------------------------------------------ | ------------------------ |
-| 🚀 **Provider Reward Boosts** (“Provider Drops”) | **6.6%**                 |
-| 🔥 **Range of Burn Rate**                        | **0 - 6.6%**             |
+You can track Provider Drops, LAVA burn and revenue: burn.lavanet.xyz.
```

Comment:
Super tiny thing, but can make it a clickable link to make it easier to navigate? For example:

You can track Provider Drops, LAVA burn and revenue: [burn.lavanet.xyz](https://burn.lavanet.xyz/).